plugins/defaultplugins/common/model/rpc/rpc.proto

```diff
@@ -80,6 +81,8 @@ message ResyncResponse {
 }
 
 
-// Response to notification
+// Response to 'from index' request. Returns indexed notification.
 message NotificationsResponse {
+    string index = 1;
```

Comment:
Index should be `uint32` type.

Reply:
Yes, it should. But for some reason, grpc cannot unmarshal such a data - there is an error during communication: proto: bad wiretype for field interfaces.NotificationsResponse.IfStats: got wiretype 0, want 2.
It seems the problem is the datatype, it started working after the change to string. I'm still looking for some solution.

Reply:
Are you recompiling both client and server on changes? I assume the error you mentioned is reported by the example client that consumes the notifications, right?

Reply:
Just tested this locally. And it works. Please ensure that you regenerated the sources and recompiled both client and server after this change.